src/components/Karma/CircleGradient.tsx

```diff
@@ -29,7 +29,7 @@ export const CircleGradient = (props: {
         width: props.size,
         height: props.size,
         justifyContent: 'center',
-        alignItems: 'center',
+        alignItems: 'baseline',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the `alignItems` property from 'center' to 'baseline' could significantly alter the layout of your components. Make sure this change doesn't unintentionally misalign any elements within this container.
```diff
-        alignItems: 'center',
+        alignItems: 'baseline',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->